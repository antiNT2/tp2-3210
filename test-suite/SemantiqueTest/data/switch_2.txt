num a;
bool b;
enum A {
    B, C, D
}
A var;

switch(b){
    case 0:
        a = 3;
        break;
    case 1:
        a = 4;
        break;
}
