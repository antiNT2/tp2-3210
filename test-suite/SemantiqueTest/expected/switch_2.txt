Invalid type in switch of Identifier b